dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/BlockingFunction/BlockingFunction.mo

```diff
@@ -37,7 +37,9 @@ protected
   Types.VoltageModulePu UFilteredPu(start = U0Pu) "Filtered voltage module in pu (base UNom)";
 
 equation
-  UFilteredPu + tFilter * der(UFilteredPu) = UPu;
+  firstOrder.u = UPu;
```

Comment:
Why this ?

Reply:
somehow its not working and this generate a difference in the voltage response, when replacing it with the firstOrder block it gives same results as Eurostag !

Reply:
Outdated